driver-api/src/main/java/io/openmessaging/benchmark/driver/BenchmarkDriver.java

```diff
@@ -64,4 +77,47 @@ CompletableFuture<BenchmarkConsumer> createConsumer(
         String topic,
         String subscriptionName,
         ConsumerCallback consumerCallback);
+
+    class TopicInfo {
+        private final String topic;
+        private final int partitions;
+
+        public TopicInfo(String topic, int partitions) {
```

Comment:
We could use Lombok `@Data` and `@AllArgsConstructor` to avoid all the methods below

Reply:
Agreed. I didn't see lombok on the classpath so went with the above. Happy to introduce it.